queue_services/entity-filer/src/entity_filer/worker.py

```diff
@@ -338,6 +338,7 @@ async def process_filing(filing_msg: Dict, flask_app: Flask):  # pylint: disable
 
             business_type = business.legal_type if business else filing_submission['business']['legal_type']
             filing_submission.set_processed(business_type)
+            business.last_modified = filing_submission.completion_date
```

Comment:
Aren't there other events (other than filing submissions) that could affect a business' state?

See example issues in spike ticket, bcgov/entity#24950.